### DISCUSSION TITLE
How to support site level common script functions

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#16753] [Feature Requests]
hi,all:

We already have the script operation in the flows, but can not make a common script as a common SDK across all flows in currently. 

```
//the CommonSDK is the operation name.
ret = CommonSDK.my_custom_complex_math_logic_here(params);
```
I am thinking of adding a new operation called `Ghost` ref to an existing operation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
